doc/sphinxman/source/dft.rst

```diff
@@ -359,11 +359,20 @@ Note that if you are running an unrestricted computation, you should set the
     set reference uks
     energy('b3lyp')
 
-The functional may also be manually specified by calling ``energy`` with a ``dft_functional``
-argument::
+The functional may also be manually specified by calling ``energy`` (or any other keyword) 
+with a ``dft_functional`` argument::
 
     energy('scf', dft_functional = 'b3lyp') 
 
+Another alternative is providing a specially crafted `dict`-ionary to the ``dft_functional``
+argument::
+
+    custom_functional = { ... }
```

Comment:
I'd expand this line a bit to something like `custom_functional = {"name": "my_dft_soup", ...}` just to hit ppl over the head with Python Dictionary, not Psithon-style `molecule {...}`.